pandas/core/frame.py

```diff
@@ -1394,13 +1394,12 @@ def to_gbq(self, destination_table, project_id=None, chunksize=None,
         read_gbq : Read a DataFrame from Google BigQuery.
         """
         from pandas.io import gbq
-        return gbq.to_gbq(
-            self, destination_table, project_id=project_id,
-            chunksize=chunksize, reauth=reauth, if_exists=if_exists,
-            auth_local_webserver=auth_local_webserver,
-            table_schema=table_schema, location=location,
-            progress_bar=progress_bar, credentials=credentials,
-            verbose=verbose, private_key=private_key)
+        gbq.to_gbq(self, destination_table, project_id=project_id,
```

Comment:
Would be good to take this all the way through. `pandas.io.gbq.to_gbq` also has a return statement, but if that actually doesn't return anything should be removed there as well